ocs_ci/deployment/deployment.py

```diff
@@ -623,19 +625,19 @@ def deploy_ocs_via_operator(self):
 
     def deployment_with_ui(self):
         """
-        This method will deploy OCS with openshift-console UI test.
-        """
-        logger.info("Deployment of OCS will be done by openshift-console")
-        ocp_console = OpenshiftConsole(
-            config.DEPLOYMENT.get("deployment_browser", constants.CHROME_BROWSER)
-        )
-        live_deploy = "1" if config.DEPLOYMENT.get("live_deployment") else "0"
-        env_vars = {
-            "OCS_LIVE": live_deploy,
-        }
-        ocp_console.run_openshift_console(
-            suite="ceph-storage-install", env_vars=env_vars, log_suffix="ui-deployment"
-        )
+        This method will deploy OCS with openshift-console vi UI.
+        """
+        setup_ui = login_ui()
+        deployment_obj = DeploymentUI(setup_ui)
+        deployment_obj.mode = "internal"
+        deployment_obj.storage_class_type = "thin_sc"
+        deployment_obj.osd_size = "0.5T"
+        deployment_obj.is_encryption = False
+        deployment_obj.is_wide_encryption = False
+        deployment_obj.is_class_encryption = False
```

Comment:
There is already configuration parameter `encryption_at_rest` ([conf/README.md](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md)), isn't it related to some of those variables? (if yes, wouldn't be better to load the value from that configuration)